### ISSUE TITLE
CONTROVERSIAL TOPIC:  Balancing the New Characters

### ISSUE THREAD

**Issue by JCOnyx** [TheOriginalSINe/Project-M-CC#73]
"Mission Statement: This build is intended to finish the work of the PMDT's development build and get a final, unofficial build. In order to preserve the 3.6 meta, there are no rebalances to characters that are already in place, except for bug fixes. To that end, this build is based off of 3.6. At the very least all the bugs should be fixed and Knuckles and Lyn will be finished."

This is quoted directly from our README.  I've seen multiple people call out for changes to Knuckle's and Lyn's moves/properties and some say that we should keep everything as close to the leaked builds as possible.  The question is, does "Kunkles and Lyn will be finished" entail balancing them to the rest of the 3.6 cast?

I'm of the opinion it is within the scope of this project, but it shouldn't be a priority until the rest of the cosmetic additions from the leaked builds and bugfixes are applied to the 3.6 cast.  How we consider and agree upon balancing decisions is probably an important issue to discuss though, or if we should even be balancing them in the first place.

**Comment by ghost** [comment]
I personally think that we should balance them to the rest of the cast, because only then would they be globally considered to be complete and natural additions to said cast. Also, we've been told that as far as recent events go, we'll be safe as long as we don't push for this to actually be the only build that is used in tournaments on stream(for now).

**Comment by ewmb7701** [comment]
I left it purposefully vague when I created the ReadMe because I was unsure what the community wanted. I personally believe that they should be balanced. Lyn in particular has some jankiness to her. Namely, her arrows and back air.

**Comment by ghost** [comment]
As far as Lyn goes, I think that her Neutral B's function should remain the same, but the actual hitbox could use some adjustments. With her back air, I like the previous suggestion of 20-23(22 maybe?) frames of landing lag.

**Comment by bb010g** [comment]
I'd be for balancing, as long as we keep on leaving the 3.6 meta as is. Balance never hurt casual PM, and their balance is what the PMDT would have strove for. Once we finish the movesets, let them be for a while, and balance slowly and deliberately off of community feedback. As long as we keep away from a buff/nerf cycle sort of thing, we should be fine.

**Comment by oXMugetsuoXo** [comment]
+1 to balancing the new characters. My current thoughts so far:
Lyn - Bair auto cancel, Newly added Vantage(Down B) too easy to land "critical" and not enough endlag to punish poorly spaced uses. Bow(neutral B) too strong for its speed.

Knuckles - Wall climbing duration is a bit too long. Someone can edgehog by using climb on edge until opponent is close enough to recover and then let go of climb for instant ledge grab.
Knuckles can also use Grounded Glide(SideB) to quickly slide off platforms. Its very quick and safe. My suggestion would be to make it so that Grounded Glide cannot slide off platforms/edges. 

Buffing wise knuckles has no point in using his forward throw when downthrow accomplishes almost the same goal. His Backthrow also could use some help as it serves no purpose other than tossing someone off stage if youre facing the wrong direction. His back throw imo should be as strong as marios.

[This is more of a personal thing  but knuckles gives off the feel of a boxer. Would be nice to replace his current spotdodge(currently same as sonic)animation with something similar to his crouching position.]

Wish I could contribute in some way other than my own thoughts but just would like to say thnx to all of you for doing this CC.

**Comment by ghost** [comment]
I have an alternate spot dodge animation for Knuckles(from the same source as the taunt and win animation fixes I put together). I'll fix the dreads on it and see if you guys like it.

**Comment by JCOnyx** [comment]
I've been compiling a list for things that could possibly be looked at for Lyn:
1. Bair Landing Lag.  A quick check in BB and OpenSA2 shows LandingAirB does in fact have an animation, it's just not working for some reason.
2. nB startup/endlag and knockback.  Everything about the arrows could use a bit of refining.  The startup and endlag could both be increased by a few frames, as just firing an arrow after a forward or down throw has almost no consequences and is pretty good at covering a ton of tech options.  The move itself just doesn't have a lot of commitment.  The KBG seem a bit over the top, I'd suggest this as the biggest target for a nerf. BKB is actually not that bad but could be slightly lower for less potent early percent gimps.
3. DSmash sword hitbox.  This attack moves pretty far into the Z Axis, causing it to miss lower opponents on occasion when it seems it would hit.  The hilt hitbox doesn't seem to have as much of a problem.  The animation could be touched up to prevent this, or we could simply adjust the tip hitbox location.
4. Dthrow knockback.  To be honest this kill throw is just a bit too powerful with it's BKB at 6E and KBG at 28.  We'd need to discuss exactly how to nerf it, like swapping values so it has a higher KBG than BKB like most kill throws, or balancing them out like a Zelda BThrow.
5. Vantage.  The sweetspot is a little too easy to connect with and does too much damage.  It's also a bit hard to punish on shield because of said damage leading to too much shieldstun when spaced properly.  Damage should be lowered from 24 to somewhere around 16-20, which is more in line with the leaked footage.  KB should be weakened as well so it doesn't kill onstage by itself around 100%.  If after all of this the move is still too difficult to punish adding some more frames of endlag would probably suffice.

**Comment by ghost** [comment]
I'm all for balancing the new characters eventually. However, I feel that we should fix all bugs and glitches, as well as clean up the new character animations first. Then we can release it, see what the community thinks of the new characters, and make adjustments from there.

**Comment by Hellraz0r** [comment]
I'm wary of balancing decisions this early. Can we flesh out the characters more before we move there? Like, I think we should be debating Lyn's moveset compared to the leaked video before we decide its proper numbers, if we get to that point.

**Comment by jgd124** [comment]
@JekyllMD 

I agree with JekyllMD. I'm not sure we even have the frame data for the new characters yet, I'm leery of deciding anything major before then.

**Comment by Eltrion** [comment]
Agreed. Fix now characters, very minor balance changes if any. Get input from community/high level players, then balance as needed. With a possible exception for what we can see of Lyn that is different from the leaked video.

**Comment by bb010g** [comment]
:+1: for waiting on balance. We can fix the obvious problems now (Lyn bair, Isaac uspecial), but the rest should be tested for a while.

**Comment by ghost** [comment]
Another vote for waiting. These characters have been out for what, 3 days now? It would be incredibly fraudulent to claim we understand what state of balance they are in at this point. Especially given that no one here is a top level player (I know you do well Eltrion, no offense meant, but you know what I mean)

The few moves I can think of that are in a bad enough state to call a "bug" and justify a change to frame data are Lyn's Down B, Bair, Up and Side B (ledge grab box is not consistent with the rest of the cast and makes it impossible for her to sweet spot) and maybe neutral B (time will tell on this one)

**Comment by JCOnyx** [comment]
Well, I am close to top 10 PR in my region and play with ePG Chevy and HCC all the time, considered some of the top Samus and Ganon PM players respectively.  Take that as you will, but I think I'm slightly qualified to worry about current balance and how it can be handled in the future.

Also, everything you just listed for Lyn are the only things people really want to be looked at anyways besides my DThrow and DSmash comment, the latter only being for an odd animation oversight causing the move to miss randomly.

**Comment by zombiejerky** [comment]
I wish they would do as much as they want to with this, I love you guys so much. Thank you for your progress so far

**Comment by ghost** [comment]
@JCOnyx I was echoing the sentiment most of the recent comments have had. Also my bad, I didn't recognize your tag. I just knew Eltrion because he's from a nearby region. Even so, balance is subjective and decisions on it should come from a large amount of testing and a number of high level opinions, neither of which we have right now. So I just mean we should hold off for now.

**Comment by JCOnyx** [comment]
And I agree on that point.  It's way too early to implement everything people are suggesting, as you said we've only had the characters for about 3 days.  But I still wanted to share my concerns regardless.  

I made this topic so we can figure out how we want to go about and decide on balancing, not what needs to be balanced at this moment.

**Comment by ghost** [comment]
I believe this can be close for now. There are already issues for Lyn's Bair, Arrows, and I also believe Vantage. I will take a look at Dsmash hitboxes tonight to see if I can confirm that they are inappropriately placed. If they are, I'll create a new issue for that.

**Comment by JCOnyx** [comment]
I've already created an issue for her DSmash, so no worries.

**Comment by ewmb7701** [comment]
Individual issues created for characters and debate settled. Closing issue.
